scripts/noneq_adaptation/adapt_noneq.py

```diff
@@ -0,0 +1,174 @@
+from md.thermostat.utils import sample_velocities
```

Comment:
Perhaps a bit of an philosophical question, but what is the difference between things that go in the `scripts` dir vs `examples`?

Reply:
Not sure -- we could set a more deliberate convention here. `scripts/` seems appropriate for obtaining a numerical result we may reuse, or developing more "aspirational" features / use cases. `examples/` seems more appropriate for demonstrating intended usage of core functionality.

Reply:
After our discussion in person, it seemed that `scripts/` was for 'infrastructure' utilities rather than exploratory work. Perhaps this could be our first 'attic' item???

Reply:
Following your suggestion here https://github.com/proteneer/timemachine/pull/442#discussion_r649370902 , I've moved the reusable components into `optimize.protocol`. Will move the usage example into `tests/`, `examples/`, or a new `attic/` folder tomorrow as you prefer.

Reply:
Whatever you think is best, can see the argument for any of those.